frontend/apps/core-app/src/screens/RecipientRegistrationScreen/RecipientRegistrationScreen.container.tsx

```diff
@@ -1,10 +1,261 @@
 import * as React from 'react';
-import { FormDefinition, Record } from 'core-api-client';
+import { FormDefinition, FormType, Record } from 'core-api-client';
 
 import { buildDefaultRecord } from './buildDefaultRecord';
 import { RecipientRegistrationScreenComponent } from './RecipientRegistrationScreen.component';
 
-const getForms = async (): Promise<FormDefinition[]> => Promise.resolve([]);
+const DATABASE_ID = '9eab5b48-41f1-411a-8cf0-390dfee8ca37';
```

Comment:
you had introduced this config file, could you not move those IDs there, for convenience reasons? also makes it easier to replace them with own local IDs when reviewing